files/build_templates/sonic_debian_extension.j2

```diff
@@ -392,6 +392,10 @@ if [ -f {{service}} ]; then
     echo "{{service}}" | sudo tee -a $GENERATED_SERVICE_FILE
 fi
 {% endfor %}
+
+# Remove services that are to be disabled by default
+sudo sed -i '/sflow.service/d' $GENERATED_SERVICE_FILE
```

Comment:
We should provide abstraction in the build config to enable/disable services. Instead of directly modifying the sonic_debian_extensions.j2. this is ugly.